### ISSUE TITLE
Update layout for RP

### ISSUE THREAD

**Issue by Rick-Anderson** [aspnet/Docs#8030]
Update layout for RP

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b29734d0-2d2b-3b82-0456-785cc7f361fd
* Version Independent ID: ab07129c-a4dd-44de-45f9-4b49eff0b54c
* Content: [Layout in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/mvc/views/layout?view=aspnetcore-2.1)
* Content Source: [aspnetcore/mvc/views/layout.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/mvc/views/layout.md)
* Product: **aspnet-core**
* GitHub Login: @ardalis
* Microsoft Alias: **riande**

**Comment by daveabrock** [comment]
I could try this, if no one else is on it? What are you looking to have changed?

**Comment by Rick-Anderson** [comment]
Sounds great.
It's MVC with controllers.  I'd rewrite it with Razor Pages featured but also list MVC paths.

remove web-project-views.png and replace with listing the layout path for RP and MVC with controllers and views.

An example *_Layout.cshtml*: 
with the current RP layout file (not with individual user accounts). Note layout file is italic, not \`code fenced\` as in the article.

**Comment by daveabrock** [comment]
Great. I’ll get going on it and reach out with any questions....

**Comment by daveabrock** [comment]
Question for you @Rick-Anderson: what is the approach of RP vs. MVC? Like, in this document should it be: "Here's Option A, here's option B" or "We prefer RP because X, but you can also use MVC if you want"? Any idea of the Microsoft approach/preference?

**Comment by Rick-Anderson** [comment]
I would present the RP approach first. Follow that with the MVC approach. No need to say RP is recommended.

**Comment by daveabrock** [comment]
So this article is under the `MVC` chapter:

![image](https://user-images.githubusercontent.com/275862/46261955-6f028080-c4c0-11e8-9795-94003d54c61b.png)

Does that not confuse readers? Should this sit here? I know it's more of a general topic, but if users are browsing to learn about MVC, will the Razor Pages content be in the way?

**Comment by Rick-Anderson** [comment]
@daveabrock we should move it. We can move it in another PR. @tdykstra 	owns the TOC and will know where to move it.

**Comment by daveabrock** [comment]
OK. I'll update the document in the MVC section and we'll work with @tdykstra on where to move it. Sounds good.

**Comment by tdykstra** [comment]
RP is still "MVC", just not controllers and views, right?  So MVC ToC node wouldn't be expected to have only cv-content.  We're looking at a thorough overhaul of the ToC, but i would say this is OK for now.  If we're not consistent about differentiating RP vs. controllers-views rather than RP vs. MVC, that might be something to watch for and fix so people get used to thinking of MVC as comprising both RP and cv.

**Comment by Rick-Anderson** [comment]
>RP is still "MVC", just not controllers and views, right? 

I don't think so. Even if that was true, that's not the perception and not what you would think if you were on [MVC](https://docs.microsoft.com/en-us/aspnet/core/mvc/overview?view=aspnetcore-2.1) in the TOC.

**Comment by daveabrock** [comment]
Just to offer a non-Microsoft perspective (and a new guy here on this repo, so apologies about any assumptions I might be making), this seems really confusing. And I bet it is even more confusing to people who are coming into .NET Core fresh.

Casually explained, Razor Pages is a "slimmed down" version of Microsoft's MVC framework. But if I'm  browsing to an MVC section, expecting model-view-controller information and am told "use this, it's like MVC without the 'V' and the 'C'" I would be very, very confused and potentially frustrated. It might go a long way in differentiating between the two somewhere (if not already - and if it is, apologies) because I'm of the opinion that grouping RP with MVC would require clarification at a minimum. I know the repo has been getting some questions about whether to use MVC or RP, and that might help.

Thoughts?

**Comment by Rick-Anderson** [comment]
@daveabrock 	agreed. I think it's really confusing to customers. Tom will look at revamping the TOC so we don't need to worry now.
